spec/features/pictures_feature_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+feature 'Pictures' do
+
+  before do
+    signup(email: 'test@example.com', password: 'testtest', password_confirmation: 'testtest' )
```

Comment:
Metrics/LineLength: Line is too long. [95/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)